[2024-05-18T15:06:36.649+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag_v2.seceond_task manual__2024-05-18T14:46:14.316861+00:00 [queued]>
[2024-05-18T15:06:36.657+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag_v2.seceond_task manual__2024-05-18T14:46:14.316861+00:00 [queued]>
[2024-05-18T15:06:36.658+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2024-05-18T15:06:36.672+0000] {taskinstance.py:2191} INFO - Executing <Task(BashOperator): seceond_task> on 2024-05-18 14:46:14.316861+00:00
[2024-05-18T15:06:36.680+0000] {standard_task_runner.py:60} INFO - Started process 2082 to run task
[2024-05-18T15:06:36.682+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v2', 'seceond_task', 'manual__2024-05-18T14:46:14.316861+00:00', '--job-id', '352', '--raw', '--subdir', 'DAGS_FOLDER/frist-dag.py', '--cfg-path', '/tmp/tmpcrsvg_jb']
[2024-05-18T15:06:36.684+0000] {standard_task_runner.py:88} INFO - Job 352: Subtask seceond_task
[2024-05-18T15:06:36.749+0000] {task_command.py:423} INFO - Running <TaskInstance: our_first_dag_v2.seceond_task manual__2024-05-18T14:46:14.316861+00:00 [running]> on host 7b23a6949395
[2024-05-18T15:06:36.955+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='our_first_dag_v2' AIRFLOW_CTX_TASK_ID='seceond_task' AIRFLOW_CTX_EXECUTION_DATE='2024-05-18T14:46:14.316861+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-18T14:46:14.316861+00:00'
[2024-05-18T15:06:36.961+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-05-18T15:06:36.963+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'pwd']
[2024-05-18T15:06:36.984+0000] {subprocess.py:86} INFO - Output:
[2024-05-18T15:06:36.986+0000] {subprocess.py:93} INFO - /tmp/***tmpwsoqjjsz
[2024-05-18T15:06:36.987+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-05-18T15:06:37.032+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v2, task_id=seceond_task, execution_date=20240518T144614, start_date=20240518T150636, end_date=20240518T150637
[2024-05-18T15:06:37.096+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-05-18T15:06:37.128+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
